R/FilterState.R

```diff
@@ -530,6 +569,23 @@ FilterState <- R6::R6Class( # nolint
         }
       str2lang(ans)
     },
+    get_metadata_id = function() {
+      private$metadata_id
+    },
+    get_metadata_title = function() {
+      if (identical(private$metadata_title, character(0))) {
+        private$get_varname()
```

Comment:
Label is missing
<img src="https://user-images.githubusercontent.com/8597300/233504225-68ddf758-3a30-4cf3-a974-e76621f11c9c.png" width="50%" height="50%" />

This update will fix it:
```suggestion
        private$get_varlabel()
```